### ISSUE TITLE
NPE when importing an ELF file with uninitialized .got section

### ISSUE THREAD

**Issue by uski** [NationalSecurityAgency/ghidra#2416]
**Describe the bug**
ghidra fails to import an ELF file and produces a NullPointerException

**To Reproduce**
1. Open CodeBrowser
2. Click File / Import File...
3. Select ELF file, leave all options at their default values
4. Click OK

**Expected behavior**
ELF file is imported

**Attachments**
If applicable, please attach any files that caused problems or log files generated by the software.

**Environment (please complete the following information):**
Stack trace :

```
Error importing file: x.elf
java.lang.NullPointerException
	at ghidra.app.util.bin.format.elf.ElfDefaultGotPltMarkup.processGOT(ElfDefaultGotPltMarkup.java:170)
	at ghidra.app.util.bin.format.elf.ElfDefaultGotPltMarkup.processGOTSections(ElfDefaultGotPltMarkup.java:86)
	at ghidra.app.util.bin.format.elf.ElfDefaultGotPltMarkup.process(ElfDefaultGotPltMarkup.java:63)
	at ghidra.app.util.bin.format.elf.extend.ElfLoadAdapter.processGotPlt(ElfLoadAdapter.java:285)
	at ghidra.app.util.opinion.ElfProgramBuilder.load(ElfProgramBuilder.java:166)
	at ghidra.app.util.opinion.ElfProgramBuilder.loadElf(ElfProgramBuilder.java:100)
	at ghidra.app.util.opinion.ElfLoader.load(ElfLoader.java:151)
	at ghidra.app.util.opinion.AbstractLibrarySupportLoader.doLoad(AbstractLibrarySupportLoader.java:346)
	at ghidra.app.util.opinion.AbstractLibrarySupportLoader.loadProgram(AbstractLibrarySupportLoader.java:83)
	at ghidra.app.util.opinion.AbstractProgramLoader.load(AbstractProgramLoader.java:112)
	at ghidra.plugin.importer.ImporterUtilities.importSingleFile(ImporterUtilities.java:401)
	at ghidra.plugin.importer.ImporterDialog.lambda$okCallback$7(ImporterDialog.java:351)
	at ghidra.util.task.TaskLauncher$1.run(TaskLauncher.java:90)
	at ghidra.util.task.Task.monitoredRun(Task.java:126)
	at ghidra.util.task.TaskRunner.lambda$startTaskThread$1(TaskRunner.java:94)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)

---------------------------------------------------
Build Date: 2020-Feb-12 1149 EST
Ghidra Version: 9.1.2
Java Home: /usr/lib/jvm/java-11-openjdk-amd64
JVM Version: Ubuntu 11.0.7
OS: Linux 5.3.0-64-generic amd64
```

readelf (section names have been replaced by Xs of same length) :
```
$ arm-none-eabi-readelf -S x.elf 
There are 23 section headers, starting at offset 0x14fd82c:

Section Headers:
  [Nr] Name              Type            Addr     Off    Size   ES Flg Lk Inf Al
  [ 0]                   NULL            00000000 000000 000000 00      0   0  0
  [ 1] .XXXXXXXXXXX      PROGBITS        1f000000 ef7300 606090 00 WAX  0   0 32
  [ 2] .XXXXXXXXX        NOBITS          00200000 000160 a900000 00  WA  0   0  1
  [ 3] .XXXXXXXXX        NOBITS          0ab00000 000160 2f10000 00  WA  0   0  1
  [ 4] .XXXXXXXXXXXXXXX  NOBITS          0da10000 000160 14f0000 00  WA  0   0 32
  [ 5] .rodata           PROGBITS        0ef00000 000160 2a2b48 00   A  0   0 32
  [ 6] .eh_frame         PROGBITS        0f1a2b48 2a2ca8 02fa3c 00   A  0   0  4
  [ 7] .ARM.exidx        ARM_EXIDX       0f1d2584 2d26e4 01f658 00  AL  1   0  4
  [ 8] .XXXXXXX          PROGBITS        0f200000 2f2000 1fdeec 00  WA  0   0  1
  [ 9] .inidata          PROGBITS        0f400000 4f2000 14413c 00  WA  0   0 32
  [10] .got              NOBITS          0f54413c 63613c 100004 00  WA  0   0  1
  [11] .excVectors       PROGBITS        ffff0000 14fd3a0 000360 00   A  0   0 32
  [12] .bss              NOBITS          0f645000 63613c 54bb000 00  WA  0   0 4096
  [13] .XXXXXXXX         PROGBITS        14b00000 638000 7c891b 00  WA  0   0  1
  [14] .XXXXXXXXX        PROGBITS        152c891b e0091b 0f69cb 00  WA  0   0  1
  [15] .XXXXXXXX         NOBITS          153bf2e6 ef72e6 000006 00  WA  0   0  1
  [16] .XXXXXXXX         NOBITS          153c0000 ef72e6 45c000 00  WA  0   0 4096
  [17] .XXXXXXXX         NOBITS          1581c000 ef72e6 0ec000 00  WA  0   0 16384
  [18] .XXXXXXXX         NOBITS          c0000000 14fd390 c10000 00  WA  0   0  1
  [19] .iram             NOBITS          ffff2000 14fd800 000800 00  WA  0   0 1024
  [20] .ARM.attributes   ARM_ATTRIBUTES  00000000 14fd700 000039 00      0   0  1
  [21] .comment          PROGBITS        00000000 14fd739 000011 01  MS  0   0  1
  [22] .shstrtab         STRTAB          00000000 14fd74a 0000e0 00      0   0  1
Key to Flags:
  W (write), A (alloc), X (execute), M (merge), S (strings), I (info),
  L (link order), O (extra OS processing required), G (group), T (TLS),
  C (compressed), x (unknown), o (OS specific), E (exclude),
  y (purecode), p (processor specific)

```

**Comment by uski** [comment]
Offending section is : 
` [10] .got              NOBITS          0f54413c 63613c 100004 00  WA  0   0  1`

If I remove it with objcopy -R, I can then import the file successfully

**Comment by dev747368** [comment]
Can dup.  Since the .got is uninitialized, your workaround probably isn't going to negatively affect the results.  I'm going to update the title of the issue a little bit.

**Comment by ghidra1** [comment]
Fixed for 9.2.3 release
